core/src/main/java/org/infinispan/commands/remote/ClusteredGetCommand.java

```diff
@@ -78,24 +79,23 @@ public void initialize(InvocationContextFactory icf, CommandsFactory commandsFac
    public CompletableFuture<Object> invokeAsync() throws Throwable {
       // make sure the get command doesn't perform a remote call
       // as our caller is already calling the ClusteredGetCommand on all the relevant nodes
-      long flagBitSet = EnumUtil.bitSetOf(Flag.SKIP_REMOTE_LOOKUP, Flag.CACHE_MODE_LOCAL);
+      // CACHE_MODE_LOCAL is not used as it can be used when we want to ignore the ownership with respect to reads
```

Comment:
The flag forces the operation to load non-owned entries, even if it is null. When a ClusteredGetCommand comes to a node that is no longer owner of given entry, but the topology id matches, `UnsuccessfulResponse.INSTANCE` is returned instead. I can't recall when this happens, but I think I've seen that in some stress test.